core/git_command.py

```diff
@@ -127,8 +188,27 @@ def git(self, *args,
                                  cwd=working_dir,
                                  env=environ,
                                  startupinfo=startupinfo)
-            stdout, stderr = p.communicate(
-                (stdin.encode(encoding=stdin_encoding) if encode else stdin) if stdin else None)
+
+            original_stdin = stdin
+            if stdin is not None and encode:
+                stdin = stdin.encode(encoding=stdin_encoding)
+
+            if show_panel and live_panel_output:
+                util.log.panel("")
+                if savvy_settings.get("show_stdin_in_output") and stdin is not None:
+                    util.log.panel_append("STDIN\n{}\n".format(original_stdin))
+                if savvy_settings.get("show_input_in_output"):
+                    util.log.panel_append("> {}\n\n".format(command_str))
+                wrapper = LoggingProcessWrapper(p)
+                stdout, stderr = wrapper.communicate(stdin)
+            else:
+                stdout, stderr = p.communicate(stdin)
+                if show_panel:
```

Comment:
also show stdin when `show_stdin_in_output` is True.

Reply:
What are you pointing at here?

I check `show_input_in_output` and the line below. And I don't want to show the stdin if you don' want to see stdout.

Reply:
I was thinking that we should insert
```python
if savvy_settings.get("show_stdin_in_output") and stdin is not None:
                   util.log.panel_append("STDIN\n{}\n".format(original_stdin))
```
to here

Reply:
Now I get it, and yes we should.